README.md

```diff
@@ -1,15 +1,42 @@
 # WELCOME TO LUNCH BUDDIES!!!
-### _An application that lets you find and match with like-minded students on campus to study, have a meal with, and more!_
----
+## Connect. Eat Lunch. Make Friends.
+
 # Table of Contents
-1. [Install](#install)
-2. [Usage](#usage)
-3. [Debugging](#debugging)
-4. [Contributing](#contributing)
+1[Description](#description)
+2[Install](#install)
+3[Usage](#usage)
+4[Debugging](#debugging)
+5[Contributing](#contributing)
     - [Front End](#front-end)
     - [Tips](#tips)
+6[License](#license)
 
 ---
+## Description
+Are you a university student looking to expand your social circle, network with like-minded individuals, or simply make new friends over a meal? Look no further than Lunch Buddies, the ultimate app designed to bring students together over lunchtime.
+
+Lunch Buddies is more than just a lunch meetup app; it's a platform for forging meaningful connections, whether you're pursuing your academic dreams, exploring new career paths, or seeking lifelong friendships.
+
+Key Features:
+Discover Lunch Partners: Browse through profiles of fellow students from various fields of study and interests.
+Schedule Lunches: Arrange meetups at your convenience, allowing you to make the most of your break time.
+Expand Your Network: Connect with individuals in your field of study, fostering opportunities for future collaborations and mentorships.
+Enhanced Social Experience: Share your interests, hobbies, and career aspirations to find like-minded individuals.
+Safe and Secure: Your privacy and safety are our top priorities. Enjoy a secure environment for making connections.
+How It Works:
+Create Your Profile: Sign up and build a personal profile highlighting your interests, major, and availability.
+Find Lunch Partners: Browse profiles and connect with students who share your interests or career goals.
+Schedule a Lunch: Propose meetups and set the date, time, and location for your lunch get-togethers.
+Meet and Connect: Enjoy meaningful conversations and make lasting connections with your lunch buddies.
+Why Lunch Buddies?
+Networking Opportunities: Expand your network and gain insights into your chosen field of study or career path.
+Friendship Building: Forge friendships that extend beyond your lunch meetups.
+Campus Community: Strengthen the sense of community among your fellow students.
+Personal Growth: Step out of your comfort zone and enhance your social skills.
+Join Lunch Buddies today and start building connections, one meal at a time. Whether you're an undergraduate, graduate student, or simply hungry for friendship, we're here to help you find your lunchtime companions.
```

Comment:
An empty line is needed between each line, otherwise GitHub's Markdown viewer will show this entire block as one big paragraph with no line breaks